src/server/package.json

```diff
@@ -22,10 +22,9 @@
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "helmet": "^3.23.1",
+    "knex": "^0.21.1",
```

Comment:
Can you give a brief description of this module and why do we need it?

Reply:
> A lot files are deleted in this PR. This is not part of current task. This creates confusion while reviewing.
> 
> Please make changes to only those files which are part of task.

My forked repo was not updated and this PR picked up changes already present in the base branch. I have merged the base branch to my fork. Now it shows relevant changes only. Can we go ahead with this one?